code/_onclick/click.dm

```diff
@@ -126,6 +126,10 @@
 	if (!isturf(loc))
 		return
 
+	if(A.Adjacent(src) && src.a_intent == INTENT_GRAB && !W)
```

Comment:
Since this is hot code, please put the adjacent check into a var so we can just check the var rather than calling Adjacent again on line 134/138.

Ideally we correct all the single var usage too, but most of the proc's code would need to change to accomplish that.

Reply:
Done